### PR TITLE
added modifications for protein-DNA basic tutorial

### LINKED ISSUE #657: small changes to the protein dna basic tutorial
- add an explanation to the restraints (the paper they come from, a bit about the syntax)
- add an explanation to the change in the dieletric constant
- change DNA to use chain C instead of B
- add updated (more readable) restraints file
- use FCC clustering rather than RMSD clustering
- add a small preprocessing step to select the first element of the ensemble (possibly removing the disordered region) using pdbtools to trim away everything after LYS66

### PR DESCRIPTION
**Pull request by mgiulini**
Closes #657

### CODE REVIEW

File: education/HADDOCK24/HADDOCK24-protein-DNA-basic/index.md [hunk 1]
```diff
@@ -126,6 +125,159 @@ Cro distorts the normal B-form DNA conformation: the OR1 DNA is bent (curved) by
...
+
+<br>
+
+<a class="prompt prompt-question">
+You can repeat the same analysis for the DNA molecule (OR1_unbound.pdb) using the residues (13+14+15+16+17+18+22+23+24+25) that are supposed to interact with one of the protein chains. What do you observe?
+</a>
+
+<hr>
+
+## Extracting a single structure from the NMR ensemble using PBD-tools (optional)
```
Comment (amjjbonvin) [inline education/HADDOCK24/HADDOCK24-protein-DNA-basic/index.md:239]: Note that you could simply add the pdb_mkensemble command as last part of the pipeline and you would then get the trimmed ensemble. May-be better? Or at least add a note for this

File: education/HADDOCK24/HADDOCK24-protein-DNA-basic/index.md [hunk 2]
```diff
@@ -74,9 +74,9 @@ An elaborate discussion of all the concepts of protein-DNA docking and the vario
 Where needed, some concepts will be explained throughout the tutorial to maintain the readability and flow of this document.
 The tutorial ends with a reference block that provides links to in depth literature or Internet documentation about various topics. 
 
-### What if I do not find an answer to my questions?
+### What if I do not find an answer to my ions?
 
-It is always possible that you have questions or run into problems for which you cannot find the answer in the regular documentation. Here are some additional links that you can find answers to your questions:
+It is always possible that you have ions or run into problems for which you cannot find the answer in the regular documentation. Here are some additional links that you can find answers to your questions:
```
Comment (mgiulini) [inline education/HADDOCK24/HADDOCK24-protein-DNA-basic/index.md]: @amjjbonvin funny typo here ;)

**Review by amjjbonvin — APPROVED** [review]
Made some edits, added a few additional questions. Check the one comment

**Comment by mgiulini** [comment]
@amjjbonvin time to merge this one I would say! I added the note you suggested about the trimmed ensemble

**Pull request merged**